app/models/conversion_host.rb

```diff
@@ -229,4 +230,10 @@ def tag_resource_as_disabled
   def default_name
     self.name ||= resource&.name
   end
+
+  # Set the default address to the first IP address associated with the resource.
+  #
+  def default_address
+    self.address ||= self.ipaddress
```

Comment:
I'm going to NAK based on this, we should be able to handle address not being set and this will break if the vm/host ipaddress changes.

Reply:
I guess the thing to do then is to not make the ipaddress field mandatory, though it makes me wonder why we have it at all then.

Reply:
Why we have which field?  `:address`?

Reply:
Yes. Seems like it should just be a virtual attribute. But, whatever, that's neither here nor there for now I suppose.